src/javascripts/ng-admin/Crud/list/ListController.js

```diff
@@ -46,10 +46,21 @@ define(function () {
         progression.start();
 
         this.ReadQueries
-            .getAll(this.view, page, true, this.search, this.sortField, this.sortDir)
-            .then(function (nextData) {
+            .getAll(this.view, page, this.search, this.sortField, this.sortDir)
+            .then(function (response) {
                 progression.done();
-                self.entries = self.entries.concat(nextData.entries);
+                var references = self.view.getReferences();
```

Comment:
You also need to retrieve references and store them into the datastore.

See https://github.com/marmelab/ng-admin/blob/a7cc56da34ed9f249df003128594124e9b2b632e/src/javascripts/ng-admin/Crud/routing.js#L73-L118.